### ISSUE TITLE
Stremio WebOS Hosted Web App

### ISSUE THREAD

**Issue by kubo6472** [Stremio/stremio-features#654]
**Is your feature request related to a problem? Please describe.**
Moving arrow navigation of a WebOS web browser is a PITA

**Describe the solution you'd like**
An IPK packaged, remote arrow keys navigatable WebOS app, that can be submitted either to webosbrew store, or to LG content store directly

**Describe alternatives you've considered**
Using the web app, which is a pain

**Additional context**
Continuation of #462

**Comment by Leyous** [comment]
I agree, we would really appreciate an LG WebOS App.

**Comment by TRtomasz** [comment]
This is repo for feature request, out of the scope.

**Comment by kubo6472** [comment]
I have trouble understanding what is the place for these kinds of feature requests then? It is a feature... A supported platform. Or am I mistaken?

**Comment by jaruba** [comment]
features can only refer to features of apps that are already available, not non-existant apps

expanding to other platforms is one of the highest Stremio priorities, you do not need to make requests of any sort for the app to be available on a new platform

if we are working on such an app or not already is not public information, as we do not tease or announce anything in advance, this is a strong Stremio internal rule, if/when it becomes available it will be announced on our official blog

Stremio has another strong rule about always trying to release to the official app stores before ever attempting to release in any unofficial store or for sideloading purposes

in short: requests for supporting new platforms are not required, which is why there is no dedicated place for them

my suggestion to users: be patient, if it comes it will come, this project is a non-profit there is no reason to become dramatic or impatient about anything related to Stremio

**Comment by kubo6472** [comment]
I bought a CCWGTV (Chromecast 4 with Google TV) just for stremio. And believe me, I dealt with LG Content Store once, we have an app there. I also followed along Jellyfin's endeavors to get into the official LG content store. If you're working on it, I wish you the best of luck. By your response I can safely assume this was brought to the attention of Stremio employees now, and is being dealt with, one way, or the other.

**Comment by jaruba** [comment]
Stremio is now available on 2020+ LG TVs: https://blog.stremio.com/stremio-is-now-available-on-lg-tvs-for-models-2020/

**Comment by nikosT** [comment]
What about 2019 models?

**Comment by jaruba** [comment]
@nikosT it will never come to 2019 models, the technologies available on 2019 and older LG TVs are just ridiculously old (much older than the tech available in 2019) which would make the app very hard to maintain
